bigchaindb/core.py

```diff
@@ -127,11 +129,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
 
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
```

Comment:
We need to be careful when using r.now because this is actually a query to the database to get the database server time. Besides the extra delay for the query I don't know how this values differs from time(). Not sure if rethinkdb uses the clock of the machine it is working on or if uses some kind of synchronized clock when in a cluster